stopcovid/sms/message_log/types.py

```diff
@@ -12,11 +13,13 @@ class LogMessageCommandType:
 class LogMessageCommand:
     command_type: str
     payload: dict
+    approximate_arrival: datetime
 
 
 class LogMessageCommandSchema(Schema):
     command_type = fields.Str(required=True)
     payload = fields.Dict(required=True)
+    approximate_arrival = fields.DateTime(required=True)
```

Comment:
to avoid breaking in production, I think you'll need to make this field optional. Or you need to release the writing components before the reading components so that you *know* the field is populated.

Reply:
I'm wrong. Ignore.